components/FeatureHighlight/tests/unit/FeatureHighlightTypographyThemerTests.m

```diff
@@ -0,0 +1,56 @@
+/*
+ Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import "MaterialFeatureHighlight.h"
+#import "MDCFeatureHighlightTypographyThemer.h"
+
+#import <XCTest/XCTest.h>
+
+@interface FeatureHighlightTypographyThemerTests : XCTestCase
+@property (nonatomic, strong) UIView *highlightedView;
+@property (nonatomic, strong) UIView *showView;
+
+@end
+
+@implementation FeatureHighlightTypographyThemerTests
+
+- (void)setUp {
+  [super setUp];
+  self.showView = [[UIView alloc] init];
+  self.highlightedView = [[UIView alloc] init];
+  [self.showView addSubview:self.highlightedView];
+}
+
+- (void)tearDown {
+  [super tearDown];
+  self.showView = nil;
+  self.highlightedView = nil;
+}
+
+- (void)testFeaturehighlightTypographyThemer {
+  MDCFeatureHighlightViewController *featureHighlightViewController =
+      [[MDCFeatureHighlightViewController alloc] initWithHighlightedView:self.highlightedView
```

Comment:
got rid of adding the subview, init is not allowed for this class and the other initilalizer tries to take snapshot after update which crashes for this test, since there is no view on the display.